values.yaml

```diff
@@ -25,6 +23,8 @@ const:
     image: docker.sourcegraph.com/query-runner:2.10.0
   repoUpdater:
     image: docker.sourcegraph.com/repo-updater:2.10.0
+  indexedSearch:
+    image: docker.sourcegraph.com/zoekt:18-05-30_3d2275e
```

Comment:
Why isn't this tagged, as well?

Reply:
Our zoekt fork is not update on the same release-schedule as all of the other images above it in this list.

Reply:
Oops, I missed that this was zoekt and just read `indexedSearch`. That makes sense for this one. Thanks!